flyte-install/index.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "Flyte",
+  "description": "Flyte Installation",
+  "details": {
+    "steps": [
+      {
+        "code" : "set-env.sh",
+        "title": "Step 1",
```

Comment:
lets make multiple steps, where each step does one thing. this may help hide the time and rename them too

So 
Step 1 -> Initialize a kubernetes environment
             Description: We can talk that we are setting up a minikube environment, this will take a moment, but continue to walk through
Step 2 -> A quick overview of components (We can show the diagram of Flyte)
                         ( I will put the image here)
Step 3 -> Now that k8s cluster is ready, lets deploy flyte
               Description shell command and explain where we got the yaml from. We will also talk about sandbox cluster here
Step 4 -> Lets wait for Flyte components to spin up 
```  # wait for flyteadmin deployment to complete
    kubectl -n flyte rollout status deployment flyteadmin
    echo ""
    kubectl -n flyte rollout status deployment flytepropeller
    echo ""
    kubectl -n flyte rollout status deployment minio
    echo ""
    kubectl -n flyte rollout status deployment contour```
Step 4: Let us register some examples
              Description: Explain registration
Step 5: Let us access the UI and launch an execution from the UI
Step 6: Observe the results
Step 7: Done

Reply:
Let's move intro of flyte in katakoda intro and use step one for providing a brief idea about flyte component 

Currently content is not aligned please comment the contetnt that we needed in all steps